src/components/PublishContainer/PublishedScreen/PublishedScreen.tsx

```diff
@@ -42,6 +44,19 @@ export const PublishedScreen: FunctionComponent<PublishScreen> = ({
     };
   });
 
+  const generateZipFile = (): void => {
+    const zip = new JSZip();
+    publishedDocuments.forEach((document) => {
+      const file = JSON.stringify(document.wrappedDocument);
+      const blob = new Blob([file], { type: "text/json;charset=utf-8" });
+      zip.file(generateFileName(config, document.fileName, "tt"), blob);
+    });
+
+    zip.generateAsync({ type: "blob" }).then((content) => {
+      saveAs(content, generateFileName(config, "TradeTrust-Documents", "zip"));
```

Comment:
i think can name as just `Documents` will do? cos creator is quite universal, can be use for other stuff i think

Reply:
ok sure, will update it